From 57d357f3a8d4a80749f225bef2dc5dc99ba636cd Mon Sep 17 00:00:00 2001
From: Liu Ying <Ying.Liu@freescale.com>
Date: Wed, 7 Aug 2013 18:59:18 +0800
Subject: [PATCH 405/505] ENGR00274311 mxc fb vsync: don't include
 linux/compiler.h

This patch removes the unnecessary linux/compiler.h header
file for the mxc fb vsync unit test.

Signed-off-by: Liu Ying <Ying.Liu@freescale.com>
---
 test/mxc_fb_test/mxc_fb_vsync_test.c | 3 +--
 1 file changed, 1 insertion(+), 2 deletions(-)

diff --git a/test/mxc_fb_test/mxc_fb_vsync_test.c b/test/mxc_fb_test/mxc_fb_vsync_test.c
index 5de7205..a4852de 100644
--- a/test/mxc_fb_test/mxc_fb_vsync_test.c
+++ b/test/mxc_fb_test/mxc_fb_vsync_test.c
@@ -1,5 +1,5 @@
 /*
- * Copyright 2004-2009 Freescale Semiconductor, Inc. All rights reserved.
+ * Copyright 2004-2013 Freescale Semiconductor, Inc. All rights reserved.
  */
 
 /*
@@ -25,7 +25,6 @@
 #include <sys/fcntl.h>
 #include <stdio.h>
 #include <stdlib.h>
-#include <linux/compiler.h>
 #include <math.h>
 #include <string.h>
 
-- 
1.8.0

